db/seeds/base/add_applications.rb

```diff
@@ -1,27 +1,60 @@
-all_lead_providers = LeadProvider.all
+# frozen_string_literal: true
 
-# users with one application each
-FactoryBot.create_list(
-  :application,
-  20,
-  :with_random_user,
-  :with_random_work_setting,
-  ecf_id: SecureRandom.uuid,
-  lead_provider: all_lead_providers.last,
-  course: Course.all.sample,
-  lead_provider_approval_status: "accepted",
-  participant_outcome_state: "passed",
-)
+LeadProvider.find_each do |lead_provider|
+  [1, 2, 3, 4].each do
```

Comment:
Can we perhaps tone this down a little bit for non-review environments?

I did this on ECF with [a config file](https://github.com/DFE-Digital/early-careers-framework/blob/main/config/seed_quantities.yml) but maybe here just do something like:

```suggestion
LeadProvider.find_each do |lead_provider|
    quantity = (Rails.env.review?) ? 4 : 1
    
    quantity.times do
```

Reply:
Thinking about this further, we probably want 0 for `test`

```suggestion
LeadProvider.find_each do |lead_provider|
  quantity = { "review" => 4, "development" => 1 }.fetch(Rails.env, 0)
  
  quantity.times do
```